src/components/basic/DlButton/DlButton.vue

```diff
@@ -187,7 +187,19 @@ export default defineComponent({
         active: { type: Boolean, default: false, required: false },
         styles: { type: [Object, String], default: null },
         shaded: { type: Boolean, default: false },
-        outlined: Boolean
+        outlined: Boolean,
+        /**
+         * Overwrite default background color on hover
+         */
+        hoverBgColor: { type: String, default: '' },
+        /**
+         * Overwrite default border color on hover
+         */
+        hoverBorderColor: { type: String, default: '' },
+        /**
+         * Overwrite default text color on hover
+         */
+        hoverTextColor: { type: String, default: '' }
```

Comment:
i would set default of null instead as it shouldn't have a value at all if we do not pass any

Reply:
done